bind/map.jinja

```diff
@@ -12,7 +12,7 @@
         'default_zones_config': '/etc/bind/named.conf.default-zones',
         'logging_config': '/etc/bind/named.conf.logging',
         'rndc_client_config': '/etc/bind/rndc.conf',
-        'named_directory': '/var/cache/bind/zones',
```

Comment:
I'm comparing this to my own nameserver and currently zones are being deployed to `/var/cache/bind/zones` so, following the discussion above, this change *will* break existing setups (actually, I understand it will recreate all the zones again, now under `/var/cache/bind`, and leave a lingering `/var/cache/bind/zones` directory)

I'm OK with this. What do you think, @joe-bowman, @aanriot ?

Reply:
Indeed @javierbertoli , but I would prefer to keep the same directory to avoid problems with current users, could it be possible @joe-bowman ? Besides that, I'm ok to merge the PR.

Reply:
The debian standard is `/var/cache/bind/zone` better to stick with the distro defaults where possible.